services/README.md

```diff
@@ -0,0 +1,54 @@
+# A Quick Start Guide
+
+This guide provides instructions on how to start the SCX scheduler and check its logs.
```

Comment:
I think we should specify that the guide is specifically about how to start the scheduler as a systemd service. What do you think about this?

This guide provides instructions on how to use an SCX scheduler as a systemd service.

Reply:
Fixed (probably!)